ui/src/app/components/copy-modal.tsx

```diff
@@ -48,7 +49,7 @@ export interface CopyModalProps {
   onClose: Function;
   onCopy: Function;
   resourceType: ResourceType;
-  saveFunction: (CopyRequest) => Promise<FileDetail | ConceptSet>;
```

Comment:
this is a common gotcha!  this syntax does not actually declare the type but instead assigns a variable name which is not used